src/main/java/fi/aalto/cs/apluscourses/intellij/model/CourseProject.java

```diff
@@ -46,7 +48,8 @@
   @NotNull
   public final Event exercisesUpdated;
 
-  private volatile User user;
+  @NotNull
+  private final ObservableProperty<User> user = new ObservableReadWriteProperty<>(null);
```

Comment:
We have this convention to make these public to avoid having getters. (`getUser().set(...)` is confusing)